packages/core-data/src/entities.js

```diff
@@ -39,13 +39,15 @@ export const defaultEntities = [
 		kind: 'root',
 		key: 'slug',
 		baseURL: '/wp/v2/types',
+		params: { context: 'edit' },
```

Comment:
Should this be  something more like "defaultParams" or something like that? (Don't take my suggestion as a recommendation, I'm no good at naming things, just trying to raise a discussion point)

Reply:
Good question! I updated to `URLparams` and moved them all immediately below `baseURL` to try and make it clearer these are for constructing the endpoint URL.